Editor/EditorCore/ShapeEditor.cs

```diff
@@ -521,41 +557,43 @@ public override ProBuilderMesh Build(bool preview = false)
             }
         }
 
+        [Serializable]
         class Arch : ShapeBuilder
         {
-            static float s_Angle = 180.0f;
-            static float s_Radius = 3.0f;
-            static float s_Width = 0.50f;
-            static float s_Depth = 1f;
-            static int s_RadiusSegments = 6;
-            static bool s_EndCaps = true;
+            static Pref<float> s_Angle = new Pref<float>("ShapeBuilder.Arch.s_Angle", 180.0f);
+            static Pref<float> s_Radius = new Pref<float>("ShapeBuilder.Arch.s_Radius", 3.0f);
+            static Pref<float> s_Width = new Pref<float>("ShapeBuilder.Arch.s_Width", 0.50f);
+            static Pref<float> s_Depth = new Pref<float>("ShapeBuilder.Arch.s_Depth", 1f);
+            static Pref<int> s_RadiusSegments = new Pref<int>("ShapeBuilder.Arch.s_RadiusSegments", 6);
+            static Pref<bool> s_EndCaps = new Pref<bool>("ShapeBuilder.Arch.s_EndCaps", true);
```

Comment:
These Pref are not explicitly assigned to `SettingsScope.User` like with the rest of the shapes.